common/app/toasts/Toasts.jsx

```diff
@@ -42,7 +42,10 @@ const addDispatchableActionsToToast = createSelector(
       finalBarStyle = rightBarStyle;
     }
     const onClick = !registeredActions[actionCreator] ?
-      null :
+      () => {
+        window.open(actionCreator, '_blank');
```

Comment:
Instead of creating an onclick, this should check if the actionCreator is a link and render an anchor tag instead.